src/services/api/axios.ts

```
@@ -1,24 +1,118 @@
   baseURL: process.env.REACT_APP_API_URL,
+  withCredentials: true,
   headers: {
     'Content-Type': 'application/json',
-    // Authorization: `Bearer ${token}`,
   },
   timeout: 5000,
 });
 
+// 토큰 재발급 상태 관리
+let isRefreshing = false;
+let failedQueue: { resolve: Function; reject: Function }[] = [];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)